test/truefi2/LoanFactory2.test.ts

```diff
@@ -309,9 +309,9 @@ describe('LoanFactory2', () => {
     })
 
     describe('reverts if', () => {
-      it('caller is not CreditAgency', async () => {
+      it('caller is not CreditAgency or loan', async () => {
         await expect(loanFactory.connect(borrower).createDebtToken(pool.address, borrower.address, parseEth(1)))
-          .to.be.revertedWith('LoanFactory: Caller is not the credit agency')
+          .to.be.revertedWith('LoanFactory: Caller is neither credit agency nor loan')
       })
 
```

Comment:
Shouldn't we add a test, for a case when a loan (not necessarily a real one) calls `createDebtToken()`?

Reply:
We have a lot of these tests in LoanToken default tests. There's no really any other option to test it so this test would be identical to LoanToken tests where enterDefault is called